presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestEliminateCrossJoins.java

```diff
@@ -65,16 +64,14 @@ public void testEliminateCrossJoin()
                 .setSystemProperty(REORDER_JOINS, "true")
                 .on(crossJoinAndJoin(INNER))
                 .matches(
-                        project(
+                        join(INNER,
```

Comment:
Can you please extract changes in this file to separate commit, as it is not related either to changes in `pruneInputs` or `restrictOutputs`